### DISCUSSION TITLE
没有使用video组件打包h5时有taro-video-control_3.entry文件

### DISCUSSION THREAD

**Discussion by jianjiachenghub** [NervJS/taro#12296] [Q & A] [answered]
@tarojs/components 里一些没有使用的组件却被打包了，而且size不小，看组件是采用es module写的，并且文档上说默认会进行tree shaking 那么这些组件代码不应该被打包呀
![image](https://user-images.githubusercontent.com/41509493/184614132-08d4883e-c5a9-45a4-8237-71fabf239717.png)

**Comment by TheKonka** [comment]
同遇到这个问题

**Comment by ZakaryCode** [comment, marked as answer]
tree-shaking 不会抖掉懒加载依赖的，如果想要绝对包体缩小可以参考[文档](https://nervjs.github.io/taro-docs/docs/babel-config#dynamic-import-node)
